### ISSUE TITLE
Vision mangelhaft

### ISSUE THREAD

**Issue by sebulino** [sebulino/Strategie#18]
Die Vision sollte in einem Satz darstellen, wo wir als Partei hinwollen. Es soll kein aasdefiniertes Programm sein, sondern ein Leitlinie, ein Leuchtturm. In der jetztigen Form ist das viel zu ausführlich und ausufernd.

**Comment by Bitsammelwanne** [comment]
"Wir wollen, dass neue Techniken für alle Menschen eingesetzt werden."

Muss vermutlich noch etwas griffiger und plakativer formuliert werden ;)

Ziel:
- Technikfreundlich (Abgrenzung gegen Grüne, zumindestens bezogen auf ihr Image ;) )
- Für die Menschen nicht gegen sie (Abgrenzung gegen CDU, SPD,... mit ihren Überwachungsgesetzen)
- Für alle Menschen nicht nur für wenige (Abgrenzung gegen FDP)
- Technik Kompetenz (Abgrenzung gegen viele, z.B. gegen die Linke)

**Comment by elyday** [comment]
@sebulino Muss es tatsächlich in einem Satz dargestellt werden? Würden nicht auch bis zu 3 Sätze gehen?

**Comment by sebulino** [comment]
Nein, also theoretisch gibt es da kein "muss". Erfahrungsgemäß ist es aber prägnanter so. 
Was in meinen Augen dafür spricht, das so kurz wie möglich zu halten:
* Es sollte eher der Blick von oben sein und nicht zu sehr ins Detail gehen, und wenig Raum vermeidet.
* In einer idealen Welt sollten dann alle Piraten diese Vision vor Augen haben. Damit das klappt sollte sie möglichst eingängig sein. Und das ist umso leichter, je kürzer die Vision ist. 

Ich hab mal drei Beispielseiten, wo verschiedene Visionen (von Unternehmen leider, aber es geht ja eher um das Format) aufgeführt sind.

http://founders-playbook.de/vision-beispiele/
https://www.mehr-fuehren.de/beispiele-unternehmensvisionen/
https://www.notabout.me/2014/03/11/6-grossartige-vision-statements-und-gegen-beispiele/

**Comment by ChristianHachmann** [comment]
Ich bin mit der aktuellen Form recht unzufrieden (und, wie schon anderswo erwähnt, auch nicht überzeugt, dass in einem Strategieprozess mal "so nebenbei" griffig eine Vision formuliert werden kann):

Speziell werden ein paar konkrete Einzelpunkte genannt (und damit naturgemäß hervorgehoben).

Dass speziell von "ortsunabhängiger Arbeit" die Rede ist, befremdet mich hier etwas. Also nicht, dass das kein relevanter Punkt wäre. Aber bei dieser Detailtiefe könnte man viele weitere Punkte aufführen.

Ich kippe hier mal zwei Textschnipsel hin, die von der Detailtiefe eher passen könnten (aber eben recht unkonkret sind).
Zur Zustandsbeschreibung (der zweite Teil ist der Präambel des Grundsatzprogramms entlehnt, scheint mir aber allein zu negativ):
`Der uralte Traum, das Wissen der Menschheit Allen verfügbar zu machen, ist durch die digitale Revolution in greifbare Nähe gerückt. Zugleich sind durch die Verfügbarkeit und den Ausbau von Überwachungs- und Kontrollwerkzeugen die Würde und die Freiheit des Menschen in bisher ungeahnter Art und Weise gefährdet.`

Hier wäre ein Absatz zum Digitalen:
`Wir Piraten möchten die Potenziale der Informationsgesellschaft erschließen und eine Nutzung zu Wohle Aller ermöglichen. Dies umfasst den Zugang zu Information ebenso wie die Teilnahme an der öffentlichen Debatte, die Verbreitung eigener Inhalte und die kreative Auseinandersetzung mit den Inhalten Anderer, mithin dem kulturellen Erbe der Menschheit.`

PS: Bzgl. Pathoslevel können wir natürlich geteilter Meinung sein. Persönlich bin ich da nicht leidenschaftlich.

Ich möchte hier nur dem Versuch etwas entgegenhalten, hier inhaltliche Einzelpunkte prominent als "Vision" herauszustellen. Wenn man einfach wenige Top-Schwerpunkte will, sollten wir schlicht abstimmen, welche einzelnen Punkte uns so wichtig sind. Das Ergebnis erfüllt aber dann sicher wieder nicht die Anforderung an eine Vision.
